routes/api.php

```diff
@@ -25,6 +26,8 @@
 Route::post('/product/{id}/generateReport', [ProductController::class, 'generateReportForOneProduct']);
 Route::post('/product/generateReport', [ProductController::class, 'generateReportForExpensiveProducts']);
 Route::post('/product/generateReportChart', [ProductController::class, 'generateReportForProductStatesGraph']);
+Route::get('/download', [ProductController::class, 'download'])->name('download');
+Route::get('/deleteFile', [ProductController::class, 'deleteFile'])->name('deleteFile');
```

Comment:
Do we need this route?

Reply:
This was old code, removed it. Can you check again